### DISCUSSION TITLE
Themes (Items) - 2 basic doubts about

### DISCUSSION THREAD

**Discussion by PCO62** [Flow-Launcher/Flow.Launcher#2016] [Q&A]
Hi

I'm trying to make my own themes and I've doubts about 2 items:
- what is the **_"PendingLine"_** item?
- what is the **_"ItemGlyph"_**?
... it's just!...

**Thank you  :)**

**Comment by onesounds** [comment]
- pendingline : The color of the progress bar that is displayed when loading. 
- ItemGlyph : In the Flow's Appearance settings, you'll find the Use Glyph Icon setting. This is the color of the glyph that is displayed when this setting is enabled. There are two states: not selected and selected.

**Reply by PCO62** [reply]
> * pendingline : The color of the progress bar that is displayed when loading.
> * ItemGlyph : In the Flow's Appearance settings, you'll find the Use Glyph Icon setting. This is the color of the glyph that is displayed when this setting is enabled. There are two states: not selected and selected.

**_Hi @onesounds_**

**Thank you very much.**
**You have an impeccable team** helping and supporting FL users, even when the questions asked are very basic.

My problem is related to the concept of "Glyph" (my native language is not English - I learned it self-taught).
Previously my question, I looked up the term/concept of "Glyph" and found these definitions 
 (from WordWeb app):
>  [Noun: glyph
> >     |glyph|
> >          1. Glyptic art in the form of a symbolic figure carved or incised in relief. ~ glyptic art, glyptography
> >          2. (computing) a graphic symbol or small picture, e.g. on an icon. ~ character, graph, grapheme, graphic symbol]


**_You said_**
> "...  In the Flow's _Appearance settings_, you'll find the _Use Glyph Icon setting_..."

Excuse my ignorance,..., "_Glyph Icon setting_" doesn't appear explicitly!... Can I assume that it refers to "_**Use Segoe Fluent Icons**_" **(1)**?!... 
And, in this case, affects 
**(1a)** and **(1b)**, or just one of them, and which one?

![FL_Glyph_230331_3RedSz](https://user-images.githubusercontent.com/93612520/229170363-971f407d-0b62-48fc-8461-4bbdb5f67da7.png)


_**Thanks, again!**_ :)

**Reply by onesounds** [reply]
yeah. fluent icon is glyph. it will display in (1b). (basic plugins)

**Reply by PCO62** [reply]
> * pendingline : The color of the progress bar that is displayed when loading.

... I forgot... in this case, I understood perfectly! :)

**Reply by PCO62** [reply]
> yeah. fluent icon is glyph. it will display in (1b). (basic plugins)

Thank you :)

**Reply by taooceros** [reply]
Basically glyph means it is displayed as text (with some special font)

**Reply by PCO62** [reply]
> Basically glyph means it is displayed as text (with some special font)

**@taooceros**
Thanks for clarifying the concept! :) 👍
